### PR TITLE
client/web: add debug card to details page

### PR DESCRIPTION
**Pull request by willnorris**
Add a new "Debug" card at the bottom of the details page. It's maybe premature to add a separate card for this, since all it currently lists is whether the device is using TUN mode and (for Synology) the DSM version. But I think it may be helpful to add client connectivity data (like shown on admin console machine page) as well as a bug report button.  Those can come soon after the 1.56 launch.

Updates #10261

### CODE REVIEW

**Review by soniaappasamy — APPROVED** [review]

File: client/web/src/components/views/device-details-view.tsx [hunk 1]
```diff
@@ -151,6 +151,23 @@ export default function DeviceDetailsView({
             </tbody>
           </table>
         </Card>
+        <Card noPadding className="-mx-5 p-5 details-card">
+          <h2 className="mb-2">Debug</h2>
+          <table>
+            <tbody>
+              <tr>
+                <td>TUN Mode</td>
+                <td>{node.TUNMode ? "true" : "false"}</td>
```
Comment (soniaappasamy) [inline client/web/src/components/views/device-details-view.tsx:160]:
```suggestion
                <td>{node.TUNMode ? "On" : "Off"}</td>
```

Comment (willnorris) [reply client/web/src/components/views/device-details-view.tsx:160]: Looks like admins console uses "Yes / No" so maybe that?

Comment (soniaappasamy) [reply client/web/src/components/views/device-details-view.tsx:160]: That works!

**Comment by soniaappasamy** [comment]
Just ran this locally - the new section is a little bare, but I think that's okay. Would be a good spot to place bugreports for 1.58...

For now fine with either this new section or just adding them to the bottom of the general section.

**Pull request merged**